### PR TITLE
Add missing field auto_close_items to Task, CreateTask, UpdateTask

### PR DESCRIPTION
**Pull request by sumanrana10048483**
### Summary

`auto_close_items` field is there in official [documentation](https://developers.lokalise.com/reference/tasks-object) but Golang library don't have that field.

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here.

### CODE REVIEW

**Comment by sumanrana10048483** [comment]
@andrewLab-lokalise Please review and merge this PR. I already raised this issue to Lokalise technical support team.

**Review by yarlson — APPROVED** [review]

File: svc_task.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ type TaskService struct {
 // Service entity objects
 // _____________________________________________________________________________________________________________________
 
-//noinspection GoUnusedConst
+// noinspection GoUnusedConst
```
Comment (yarlson) [inline svc_task.go:20]: `go fmt` fails on this changes

**Review by yarlson — CHANGES_REQUESTED** [review]

File: svc_task.go (hunk 1 above)
Comment (sumanrana10048483) [reply svc_task.go]: This is fixed.

**Comment by yarlson** [comment]
@sumanrana10048483 please, run `go fmt ./...`, and commit the changes

**Comment by sumanrana10048483** [comment]
> @sumanrana10048483 please, run `go fmt ./...`, and commit the changes

I ran this command. And I get this changes automatically https://github.com/lokalise/go-lokalise-api/pull/41/commits/ed781fa1a756940032783672193177d7bd89bbcb

**Pull request merged**